FLS01/README.md

```diff
@@ -304,15 +303,15 @@ If no currency_code is specified in request:
         "currency_code": "CHF",
         "payment_options": [
           {
-            "option": "SEPA Instant",
-            "id": 2,
+            "option": "Bank transfer",
```

Comment:
The E in SEPA stands for euro, so the term does not fit to CHF.

Reply:
true,but overall  the response here is an example as this is something that will differ per provider anyway based on the fiat rails they support